### PR TITLE
PRP-3 Follow setup steps for new infra repo

### PR DESCRIPTION
**Pull request by aplybeah**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-3

## Changes
* created new infra PR
* added resources to AWS
* set up dev environment

## Context for reviewers
As a part of the platform initiative, we are using a template in order to lay the foundations for our infrastructure.

## Testing
1. Log into the wic-prp AWS account
2. navigate around the account and look at the resources that have been created

_terraform state file for the dev environment_
Screen Shot 2022-11-15 at 12 16 34 PM

_ECS service in the dev environment_
Screen Shot 2022-11-15 at 12 17 53 PM

### CODE REVIEW

File: infra/accounts/account/main.tf [hunk 1]
```diff
@@ -24,13 +24,13 @@ terraform {
...
-  #uncomment#   dynamodb_table = "<TF_LOCKS_TABLE_NAME>"
-  #uncomment#   key            = "infra/account.tfstate"
-  #uncomment#   region         = "<REGION>"
-  #uncomment#   encrypt        = "true"
-  #uncomment# }
+  backend "s3" {
+    bucket         = "wic-prp-636249768232-us-east-1-tf-state"
+    dynamodb_table = "wic-prp-tf-state-locks"
+    key            = "infra/account.tfstate"
+    region         = "us-east-1"
```
Comment (microwavenby) [inline infra/accounts/account/main.tf]: Is this a shared backend S3 bucket across Nava infra? 

I was wondering if we'd prefer a west-coast server for Montana (but if it's shared, makes sense to leave it as is)

**Review by microwavenby — COMMENTED** [review]
plan.out files shouldn't be in the repo

We should add those to the .gitignore

File: infra/accounts/account/main.tf (hunk 1 above)
Comment (rocketnova) [reply infra/accounts/account/main.tf]: I think it's just a shared bucket for all the resources specifically for PRP, I think across environments.

I think it's probably better to switch to west-coast for MT.

File: infra/app/envs/dev/main.tf [hunk 2]
```diff
@@ -6,7 +6,7 @@ locals {
   # The prefix key/value pair is used for Terraform Workspaces, which is useful for projects with multiple infrastructure developers.
   # By default, Terraform creates a workspace named “default.” If a non-default workspace is not created this prefix will equal “default”, 
   # if you choose not to use workspaces set this value to "dev" 
-  prefix = terraform.workspace
+  prefix = "dev"
```
Comment (rocketnova) [inline infra/app/envs/dev/main.tf]: @aplybeah Could we use workspaces? I think it would be helpful to me for this project to have access that.

**Review by rocketnova — COMMENTED** [review]
@aplybeah Please remove the plan.out files. I think rather than approving this one and opening another PR to address the comments, there are enough comments that we should go ahead and address them in this PR.

**Review by rocketnova — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**